servers/rendering/rasterizer_rd/shaders/scene_high_end.glsl

```diff
@@ -361,6 +361,65 @@ layout(location = 0) out vec4 frag_color;
 
 #endif // RENDER DEPTH
 
+#ifdef ALPHA_HASH_USED
+
+float hash_2d(vec2 p) {
+	return fract(1.0e4 * sin(17.0 * p.x + 0.1 * p.y) *
```

Comment:
I have a feeling this hash function is only going to look nice on NVidia cards. NVidia uses much higher precision for ``sin`` than anyone else. So hashes based on ``sin`` typically have bad artifacts on other cards.

Reply:
I checked this out and it looks fine on my AMD card. So we can forget about this until we see if it actually is an issue on any cards.